README.md

```diff
@@ -143,3 +144,30 @@ It's also possible to run the script without using docker. To this end, issue th
     export ROS_WS=/tmp/ros_ws        # define a new ROS workspace location
     mkdir $ROS_WS                    # and create it
     .moveit_ci/travis.sh
+
+## Enabling codecov.io reporting
+
+For codecov to work you need to build and link your C++ code with specific parameters.  To enable this we use the ROS package [code_coverage](https://github.com/mikeferguson/code_coverage).  Using the `code-coverage` test in your repo requires the following three changes:
+
+1. Add `<test_depend>code_coverage</test_depend>` to your package.xml
+2. Add this to your `CMakeLists.txt`:
+
+```cmake
+# to run: catkin_make -DENABLE_COVERAGE_TESTING=ON package_name_coverage
+if(CATKIN_ENABLE_TESTING AND ENABLE_COVERAGE_TESTING)
+  find_package(code_coverage REQUIRED)   # catkin package ros-*-code-coverage
+  include(CodeCoverage)
```

Comment:
I just released 0.3.0 with @rhaschke improvements - once that propagates through the build farm and gets sync'd you'll be able to drop this line

Reply:
Which line?  The find_package or the include?  Can you link me to your changes?

Reply:
The include -- see https://github.com/mikeferguson/code_coverage/pull/15 for details (again, I just did the release, it's probably 2 weeks before it syncs all the way through the build farm).

Reply:
Ah, cool.  Hopefully I'll get this merged quickly and then I can make a small PR to remove this line once your change propagates.